x-pack/plugins/security_solution/public/common/components/event_details/insights/related_alerts_by_session.tsx

```diff
@@ -99,6 +100,7 @@ export const RelatedAlertsBySession = React.memo<Props>(
         state={state}
         text={getTextFromState(state, count)}
         renderContent={renderContent}
+        extraAction={<EuiBetaBadge size="s" label={BETA} color="subdued" />}
```

Comment:
Thanks for adding the `Beta` badge.

For consistency while it's behind a feature flag, should `Related alerts by process ancestry` also have a `Beta` badge?

![related_alerts_by_process_ancestry](https://user-images.githubusercontent.com/4459398/183732804-cf60d08c-9059-45cc-8a34-7d9342e19a80.png)

Reply:
I agree, makes sense. Added here: https://github.com/elastic/kibana/pull/138419/commits/81ca4d08c0cacefa2d578bb8934b9c0c6ee4b175